spec/unit/mutest/cli_spec.rb

```diff
@@ -151,6 +151,19 @@
         let(:expected_integration) { Mutest::Integration::Rspec }
       end
 
+      context 'when specifying the default null integration explicitely' do
+        let(:flags) { %w[--use null] }
+        let(:expected_integration) { Mutest::Integration::Null }
+
+        before do
+          expect(Kernel).to receive(:require)
+            .with('mutest/integration/null')
+            .and_call_original
```

Comment:
To put it another way, this particular spec matches the one for rspec integration in the same file, while the added `and_call_original` from the other spec has already been removed.